crypto/stack/stack.c

```diff
@@ -250,7 +250,7 @@ void OPENSSL_sk_pop_free(OPENSSL_STACK *st, OPENSSL_sk_freefunc func)
         return;
     for (i = 0; i < st->num; i++)
         if (st->data[i] != NULL)
-            func(st->data[i]);
+            func((char *)st->data[i]);
```

Comment:
Because we are saying that data you put on a stack is always treated as a const....oh except for this one function where we don't. That has a bad smell to me.